cuebot/src/main/resources/opencue.properties

```diff
@@ -61,6 +61,10 @@ log.frame-log-root.default_os=${CUE_FRAME_LOG_DIR:/shots}
 # are planning to use a folder in the root, use:
 #   - log.frame-log-root.Windows=${S:}
 
+# Loki
+log.loki.enabled=false
+log.loki.url=http://localhost/loki/api
```

Comment:
Please elaborate on the docs here. I would add:
 - link to a tutorial on how to setup loki
 - quick description of what happens when this feature is on
 - what to expect if an invalid url is provided

Reply:
Have added small description